package.json

```diff
@@ -36,9 +36,11 @@
     "react": "^0.14 || ^15"
```

Comment:
Pure component is only supported from react 15.3, so peerDependencies have to be updated too